src/phase-2/scWETHv2Helper.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.13;
+
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+
+import {scWETHv2} from "./scWETHv2.sol";
+import {OracleLib} from "./OracleLib.sol";
+import {IAdapter} from "../scWeth-adapters/IAdapter.sol";
+
+// @title helper contract for just the external view methods to be used by the backend
+contract scWETHv2Helper {
+    using FixedPointMathLib for uint256;
+
+    scWETHv2 vault;
+    OracleLib oracleLib;
+
+    constructor(scWETHv2 _vault, OracleLib _oracleLib) {
+        vault = _vault;
+        oracleLib = _oracleLib;
+    }
+
+    function getDebt(IAdapter adapter) public view returns (uint256) {
+        return adapter.getDebt(address(vault));
+    }
+
+    function getCollateral(IAdapter adapter) public view returns (uint256) {
+        return oracleLib.wstEthToEth(adapter.getCollateral(address(vault)));
+    }
+
+    /// @notice returns the net leverage that the strategy is using right now (1e18 = 100%)
+    function getLeverage() public view returns (uint256) {
+        uint256 coll = vault.totalCollateral();
+        return coll > 0 ? coll.divWadUp(coll - vault.totalDebt()) : 0;
```

Comment:
```suggestion
        uint256 collateral = vault.totalCollateral();
        return coll > 0 ? collateral.divWadUp(collateral - vault.totalDebt()) : 0;
```
use full name for consistency

Reply:
[Fixed](https://github.com/lindy-labs/sandclock-contracts/pull/54/commits/e96de8d98745bb1c90b929f45255093f5c59a1ea)